.github/workflows/test-integration.yml

```diff
@@ -10,6 +10,7 @@ on:
 
 jobs:
   test-integration-up4-docker:
+    timeout-minutes: 15
```

Comment:
It looks like this edit did not have any effect because the job still fails with the same error as before:
``` bash
=== RUN   TestSliceMeter
panic: test timed out after 10m0s
```